src/main/scala/org/apache/sysml/api/ml/scala/ScriptsUtils.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.sysml.api.ml.scala
+
+import java.io.File
+
+object ScriptsUtils {
+  var systemmlHome = System.getenv("SYSTEMML_HOME")
+  def resolvePath(filename:String):String = {
+    import java.io.File
+    ScriptsUtils.systemmlHome + File.separator + "algorithms" + File.separator + filename
```

Comment:
One other issue to track here is that there is currently a disconnect with the directory structure between the Git repo and our distribution tar/zip packages.  In the former case, if `SYSTEMML_HOME` is set to the base of the Git repo (which is the common pattern), then this line needs to change to `ScriptsUtils.systemmlHome + File.separator + "scripts" + File.separator + "algorithms" + File.separator + filename` to reflect that `algorithms` is nested under the `scripts` folder.  Otherwise, the associated test would fail.  However, in the latter case, our tar/zip packages have the `algorithms` folder at the base (not nested), so this line would be correct for the API to be able to be used.